met-api/src/met_api/services/poll_response_service.py

```diff
@@ -83,6 +84,7 @@ def get_poll_details_with_response_counts(poll_id):
             db.session.query(
                 PollAnswer.id.label('answer_id'),
                 PollAnswer.answer_text,
+                # pylint: disable=not-callable
                 func.count(PollResponseModel.selected_answer_id).label('response_count')
```

Comment:
From what I've read, it's a known issue for pylint to be labelling `func` method calls as false positives for this rule. You may be able to get around the issue with this stackoverflow answer: https://stackoverflow.com/a/75789407